src/client/components/QuizAnswers/QuizAnswers.styles.css

```diff
@@ -13,12 +13,23 @@
   flex-direction: row-reverse;
 }
 
-/* Agreement type questions are aligned inline */
-.inline {
+.answers {
+  padding-top: 20px;
+  margin-top: 50px;
   display: flex;
+  flex-direction: column;
   gap: 0.6em;
 }
 
+/* Agreement type questions are aligned inline */
+.inline {
+  margin-right: calc(10vw + 49px);
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  flex-direction: row;
+}
```

Comment:
For me, the options are like this. Shouldn't they be online? 
<img width="707" alt="Capture" src="https://user-images.githubusercontent.com/39958295/120387557-5c543f80-c32a-11eb-8e7f-ac994387e085.PNG">

Reply:
Yes, there have been database updates since the options looked like this. You need to update your database (delete all the table and do the migrations and seeding again). And make your you are running the branch and not develop.

Reply:
Oh, didn't know that :) Thanks Beth